tools/archetype/binding/src/main/resources/archetype-resources/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -6,6 +6,12 @@
 
 	<!-- Sample Thing Type -->
 	<thing-type id="sample">
+		<!-- Comment this in for Things which need a Bridge to function.
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="${bindingIdCamelCase}Bridge" />
```

Comment:
I think it makes sense to add it but only if we provide the source for the resulting bridge too. Including complete handling in `ThingHandlerFactory`and so on. Maybe add a hint in the JavaDoc to delete the class if not needed.

Once we got this into out skeleton we can add more example code from the docs like discovery.

Reply:
We could copy the Thing handler class and the XML structure, but those would be nearly 1:1 copies. I'm unsure if that adds any benefit or just puzzles the user...

Reply:
No, there will be not much benefit when copying it. As I mentioned above an example discovery services as `ThingHandlerService` implementation provided by the bridge handler would be nice. But we can add it later.